jetty-server/src/main/java/org/eclipse/jetty/server/handler/BufferedResponseHandler.java

```diff
@@ -189,8 +207,6 @@ public void resetBuffer()
             _aggregate = null;
         }
 
-        ;
-
         @Override
         public void write(ByteBuffer content, boolean last, Callback callback)
         {
```

Comment:
Is the logic below correct:

`if (sc > 0 && (sc < 200 || sc == 204 || sc == 205 || sc >= 300))`

I ask because I think you should rather use `HttpStatus.hasNoBody(sc)` instead.
However, the logic of `hasNoBody()` is different.
In general, for this kind of logic you should use `HttpStatus` methods such as `isRedirection()` etc. but beware that for example 302 may have a response body.